db-meta-plugin/src/main/java/ch/ivyteam/db/meta/generator/internal/mssql/MsSqlServerSqlScriptGenerator.java

```diff
@@ -62,7 +62,7 @@ protected void generateDataType(PrintWriter pr, DataType dataType)
     switch(dataType)
     {
       case CLOB:
-        pr.append("TEXT");
+        pr.append("NVARCHAR(MAX)");
         break;
       case BLOB:
         pr.append("IMAGE");

```

Comment:
Can you check if IMAGE is also deprecated and instead VARBINARY(MAX) should be used.